src/test/ui/nll/enum-drop-access.rs

```diff
@@ -0,0 +1,51 @@
+#![feature(nll)]
+
+enum DropOption<T> {
+    Some(T),
+    None,
+}
+
+impl<T> Drop for DropOption<T> {
+    fn drop(&mut self) {}
+}
+
+// Dropping opt could access the value behind the reference,
```

Comment:
(GIven that the type `T` has no trait bounds and the type carries no closures of type `Fn(T)`, I think this comment about whether the drop could access the value is **only** true if the code were makes use of specialization. Or at least, we used to have some complicated arguments about what `Drop` on a parametric type was actually capable of.)

Reply:
But that's just me being pedantic. I have no problem with you leaving this code and comment as it stands.